client/components/google-my-business/stats-nudge.js

```diff
@@ -61,10 +62,17 @@ class GoogleMyBusinessStatsNudge extends Component {
 							) }
 						</h2>
 						<div className="google-my-business__stats-nudge-button-row">
-							<Button primary onClick={ this.props.trackNudgeStartNowClicked }>
+							<Button
+								href={ `/google-my-business/${ this.props.siteSlug }` }
+								primary
+								onClick={ this.props.trackNudgeStartNowClicked }
+							>
 								{ this.props.translate( 'Start Now' ) }
 							</Button>
-							<Button onClick={ this.props.trackNudgeAlreadyListedClicked }>
+							<Button
+								href={ `/google-my-business/${ this.props.siteSlug }` }
```

Comment:
Do we really need this `href` here? We should just close the nudge, and record that event when the user clicks that button. The user won't be redirected anywhere.

Reply:
This has not been addressed.